dev/external/flatbuffers/src/idl_gen_text.cpp

```diff
@@ -266,7 +266,7 @@ struct JsonPrinter {
           fd.value.offset);
     } else if (fd.flexbuffer && opts.json_nested_flexbuffers) {
       // We could verify this FlexBuffer before access, but since this sits
-      // inside a FlatBuffer that we don't know wether it has been verified or
+      // inside a FlatBuffer that we don't know whether it has been verified or
```

Comment:
this file is copied from the upstream FlatBuffers repo, so unless it is fixed there, will be undone again.

Reply:
the wether shall live on